.github/workflows/test.yml

```diff
@@ -104,6 +104,7 @@ jobs:
       run: |
         bash -c 'git config --global --add safe.directory /opt/esp/esp-idf && export MICRO_RDK_WIFI_PASSWORD=0 && . "$IDF_PATH"/export.sh && . "$ESP_ROOT"/export-esp.sh && make clippy-esp32'
         bash -c 'export MICRO_RDK_WIFI_PASSWORD=0 && . "$IDF_PATH"/export.sh && . "$ESP_ROOT"/export-esp.sh && cd examples && cd ../ && make build-esp32-bin'
+        bash -c 'export MICRO_RDK_USE_NVS=true && . "$IDF_PATH"/export.sh && . "$ESP_ROOT"/export-esp.sh && cd examples && cd ../ && make build-esp32-with-cred-bin'
```

Comment:
Another naive question: why do we `cd examples && cd ../`? Isn't that the same as not doing either of those steps?

Reply:
I agree, but for some reason, removing this broke stuff. I'll double-check